ep/build-velox/src/build_velox.sh

```diff
@@ -10,8 +10,8 @@ BUILD_PROTOBUF=ON
 BUILD_TYPE=release
 VELOX_HOME=
 #for ep cache
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/liujiayi771/velox.git
+VELOX_BRANCH=stddev_pop
```

Comment:
Velox PR has been merged, and this change could be reverted, thanks.

Reply:
ok.